week2/week2-HW/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="kor" />
```

Comment:
```suggestion
    <meta charset="UTF-8" />
```

`lang` 수정하려고 하시다가 여기에 잘못 수정한 것 같은데 맞을까요...???